packages/kbn-dev-utils/src/vscode_config/update_vscode_config.ts

```diff
@@ -69,13 +69,13 @@ const createObjectPropOfManagedValues = (key: string, value: Record<string, any>
 const addManagedProp = (
   ast: t.ObjectExpression,
   key: string,
-  value: string | Record<string, any> | boolean
+  value: string | Record<string, any> | boolean | number
 ) => {
-  ast.properties.push(
-    typeof value === 'string' || typeof value === 'boolean'
-      ? createManagedProp(key, value)
-      : createObjectPropOfManagedValues(key, value)
-  );
+  if (['number', 'string', 'bolean'].includes(typeof value)) {
```

Comment:
```suggestion
  if (['number', 'string', 'boolean'].includes(typeof value)) {
```

--> currently   "typescript.enablePromptUseWorkspaceTsdk": {},